pytest_tests/steps/load.py

```diff
@@ -22,6 +23,25 @@ def get_services_endpoints(
     return [service_config.attributes[endpoint_attribute] for service_config in service_configs]
 
 
+@allure.title("Stop nodes")
+def stop_unused_nodes(hosting: Hosting, used_nodes_count: int):
+    service_configs = hosting.find_service_configs(STORAGE_NODE_SERVICE_NAME_REGEX)[
+        used_nodes_count:
+    ]
+    for service_config in service_configs:
+        host = hosting.get_host_by_service(service_config.name)
+        STOPPED_HOSTS.append(host.config.address)
+        host.stop_host("hard")
+
+
+@allure.title("Start nodes")
+def start_stopped_nodes(hosting: Hosting, used_nodes_count: int):
+    for host_address in STOPPED_HOSTS:
```

Comment:
we don't need to get hosts by adresses if we store Host directly in STOPPED_HOSTS

Reply:
done